forecasting_sa/models/gluonts/GluonTSForecastingPipeline.py

```diff
@@ -6,7 +6,7 @@
 
 from gluonts.model.estimator import Estimator
 from gluonts.model.seasonal_naive import SeasonalNaivePredictor
-from gluonts.model.prophet import ProphetPredictor
```

Comment:
One of the many changes in the GluonTS [api](https://ts.gluon.ai/stable/api/gluonts/gluonts.dataset.pandas.html?highlight=from_long_dataframe#gluonts.dataset.pandas.PandasDataset.from_long_dataframe)